HelpMyStreetFE/HelpMyStreetFE/Services/Requests/FilterService.cs

```diff
@@ -30,13 +39,86 @@ public SortAndFilterSet GetDefaultSortAndFilterSet(JobSet jobSet, JobStatuses? j
                 JobSet.UserAcceptedRequests => GetAcceptedRequestsDefaultSortAndFilterSet(),
                 JobSet.UserCompletedRequests => GetCompletedRequestsDefaultSortAndFilterSet(),
 
-                JobSet.GroupShifts => new SortAndFilterSet(),
-                JobSet.UserOpenShifts => new SortAndFilterSet(),
-                JobSet.UserMyShifts => new SortAndFilterSet(),
+                JobSet.GroupShifts => await GetGroupShiftsFilterSet(user, jobStatus),
+                JobSet.UserOpenShifts => await GetShiftsFilterSet(user),
+                JobSet.UserMyShifts => await GetShiftsFilterSet(user),
                 _ => throw new ArgumentException(message: $"Unexpected JobFilterRequest.JobSet value: {jobSet}", paramName: nameof(jobSet))
             };
         }
 
+        private async Task<SortAndFilterSet> GetShiftsFilterSet(User user)
+        {
+            var locations = await _userService.GetLocations(user.ID, new System.Threading.CancellationToken());
+            var locationDetails = await _addressService.GetLocationDetails(locations);
+            SortAndFilterSet filterSet = new SortAndFilterSet
+            {
+                Locations = locationDetails.Select(ld => new FilterField<Location>() { Value = ld.Location, IsSelected = true, Label = ld.ShortName }),
+                PartOfDay = new List<FilterField<PartOfDay>>()
+                {
+                    new FilterField<PartOfDay>() {Value = PartOfDay.Morning, Label = "Morning Shifts", IsSelected = true},
+                    new FilterField<PartOfDay>() {Value = PartOfDay.Afternoon, Label = "Afternoon Shifts", IsSelected = true},
+                    new FilterField<PartOfDay>() {Value = PartOfDay.Night, Label = "Night Shifts", IsSelected = true}
+                },
+                OrderBy = new List<OrderByField>
+                {
+                    new OrderByField() {Value = OrderBy.DateDue_Ascending, Label = "Soonest"},
+                    new OrderByField() {Value = OrderBy.DateDue_Descending, Label = "Furthest Away"},
```

Comment:
Furthest away sounds like "I'm looking for shifts in Barbados."  What about "Most recently added", and order by date requested?